controllers/users.js

```diff
@@ -167,18 +167,45 @@ const getUsers = async (req, res) => {
     if (transformedQuery?.filterBy === OVERDUE_TASKS) {
       try {
         const tasksData = await getOverdueTasks(days);
-        const users = new Set();
+        const userIds = new Set();
+        const usersData = [];
+
         tasksData.forEach((task) => {
-          users.add(task.assignee);
+          userIds.add(task.assignee);
```

Comment:
what if task doesn't have assignee prop

Reply:
We are fetching those tasks to have an assignee.